web/viewer.js

```diff
@@ -154,6 +154,7 @@ function getViewerConfiguration() {
       highlightAllCheckbox: document.getElementById("findHighlightAll"),
       caseSensitiveCheckbox: document.getElementById("findMatchCase"),
       entireWordCheckbox: document.getElementById("findEntireWord"),
+      matchDiacriticsCheckbox: document.getElementById("findMatchDiacritics"),
```

Comment:
Nit: Let's move this up *one* line, since the new checkbox is placed *before* the entireWord-one in the findbar.